css/properties/contain.json

```
@@ -42,6 +33,76 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "inline-size_value": {
+          "__compat": {
+            "description": "<code>inline-size</code> value",
```

For CSS properties, we don't need to suffix the subfeatures with `_value` -- like properties and methods of interfaces, there's a natural implication that the regular subfeatures of a CSS property is a possible value.

```suggestion
        "inline-size": {
          "__compat": {
```